glue_jupyter/common/state_widgets/viewer_histogram.py

```diff
@@ -18,3 +18,5 @@ def __init__(self, viewer_state):
         self.glue_state = viewer_state
 
         link_glue_choices(self, viewer_state, 'x_att')
+
+    def axis_to_bins(self):
```

Comment:
Have you tried including a vue_ prefix on this method?

Reply:
So I know I need to, but there's nothing behind the button just yet